auth0/resource_auth0_log_stream.go

```diff
@@ -0,0 +1,341 @@
+package auth0
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+
+	"gopkg.in/auth0.v4"
+	"gopkg.in/auth0.v4/management"
+)
+
+func newLogStream() *schema.Resource {
+	return &schema.Resource{
+
+		Create: createLogStream,
+		Read:   readLogStream,
+		Update: updateLogStream,
+		Delete: deleteLogStream,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"eventbridge", "eventgrid", "http", "datadog", "splunk"}, true),
+				ForceNew:    true,
+				Description: "Type of the LogStream, which indicates the Sink provider",
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"active", "paused", "suspended"}, false),
+				Description: "Status of the LogStream",
+			},
+			"sink": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"aws_account_id": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+							ForceNew:  true,
+						},
+						"aws_region": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+						},
+						"aws_partner_event_source": {
```

Comment:
I'm not sure how much of a security risk exposure of this value would be,  but I'd err on the side of this being marked as sensitive, and I would guess the same would apply to the Azure equivalent.